napari/components/viewer_model.py

```diff
@@ -70,6 +70,8 @@ def __init__(self, title='napari', ndisplay=2, order=None):
 
         self.layers = LayerList()
 
+        self.axis_labels = [''] * ndim  # labels will be empty if data is 2D
```

Comment:
Here I don't want to see us make an extra property on the viewer model with the `axis_labels`, instead we should be passing this variable to the `Dims` constructor a couple lines above, so it now looks like 
```python
self.dims = Dims(ndim, ndisplay=ndisplay, order=order, axis_label=axis_label)
```